src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -64,58 +64,43 @@
 
 
 (defn render-block
-  [blocks {:keys [type ^js literal children]} edited-at]
+  [blocks {:keys [type ^js literal children]}]
   (case (keyword type)
     :paragraph
     (conj blocks
-          (reduce
-           render-inline
-           [quo/text]
-           children))
+          [rn/view {:style style/paragraph}
+           (reduce
+            render-inline
+            [quo/text]
+            children)])
 
     :blockquote
     (conj blocks
           [rn/view {:style style/quote}
-           [quo/text literal]]
-          (when edited-at
-            [quo/text
-             {:weight :medium
-              :style  {:font-size  11
-                       :margin-top 4
-                       :color      (colors/theme-colors colors/neutral-40 colors/neutral-50)}}
-             (str " (" (i18n/label :t/edited) ")")]))
+           [quo/text literal]])
 
     :codeblock
     (conj blocks
           [rn/view {:style (merge style/block (style/code))}
-           [quo/text (subs literal 0 (dec (count literal)))]]
-          (when edited-at
-            [quo/text
-             {:weight :medium
-              :style  {:font-size  11
-                       :margin-top 4
-                       :color      (colors/theme-colors colors/neutral-40 colors/neutral-50)}}
-             (str " (" (i18n/label :t/edited) ")")]))
+           [quo/text (subs literal 0 (dec (count literal)))]])
     blocks))
 
-(defn add-edited-tag
-  [parsed-text]
-  (update parsed-text
-          (dec (count parsed-text))
-          (fn [last-literal]
-            (update last-literal
-                    :children
-                    conj
-                    {:literal (str " (" (i18n/label :t/edited) ")")
-                     :type    :edited}))))
+(defn edited-tag
+  []
+  [quo/text
+   {:weight :medium
+    :style  (style/edited-tag)}
+   (str " (" (i18n/label :t/edited) ")")])
```

Comment:
I think we should remove the space string before opening parenthesis and use margins for spacing instead.